inventory/host_vars/staging.openfoodnetwork.org.uk/config.yml

```diff
@@ -12,3 +12,15 @@ users_sysadmin:
   - "{{ core_devs }}"
   - lindhop
   - filipe
+
+# UK Staging is currently on holiday in Canada...
```

Comment:
:joy:

Reply:
https://staging.openfoodnetwork.org.uk/

Reply:
The positioning of the logo on the maple leaf is 💯